plugins/evernote.koplugin/main.lua

```diff
@@ -139,9 +145,32 @@ function EvernoteExporter:addToMainMenu(tab_item_table)
                 checked_func = function() return self.html_export end,
                 callback = function()
                     self.html_export = not self.html_export
+                    if self.html_export then self.txt_export = false end
+                    self:onSaveSettings()
+                end
+            },
+            {
+                text = _("Export to local clipping text file"),
+                checked_func = function() return self.txt_export end,
+                callback = function()
+                    self.txt_export = not self.txt_export
+                    if self.txt_export then self.html_export = false end
                     self:onSaveSettings()
                 end
             },
+            {
+                text = _("Purge history records"),
+                callback = function()
+                    self.config:purge()
+                    UIManager:show(InfoMessage:new{
+                        text = _("History records are purged.\nAll notes will be exported again next time.\nSuggest to remove existing KOReaderClipping.txt to avoid a duplication."),
+                    })
+                end
+            },
+            {
+                text = _("  - All notes will be exported again next time"),
```

Comment:
I see, so this hint is only for the purge menu item right? In this case isn't it redundant? The info message in that callback already has the same hint.

> I do think the menu items can be optimized. I cannot quite tell the benefit of the two switches to control the output destination. Why cannot we just use 'export to evernote' 'export to html' and 'export to txt'?

Yes, I agree :)

> We can decide whether exporting all documents or current document based on isDocless(), I.e. exporting all in file manager, and exporting current document in reader.

The kpvbooklet users will probably never use filemanager. We should disable export current book item using isDocless() in filemanager though.